lib/aptible/cli/helpers/tunnel.rb

```diff
@@ -23,40 +23,25 @@ module Helpers
                     end
 
       class Tunnel
-        def initialize(env, ssh_cmd)
+        def initialize(env, ssh_cmd, socket_path)
           @env = env
           @ssh_cmd = ssh_cmd
+          @socket_path = socket_path
         end
 
         def start(desired_port = 0)
           @local_port = desired_port
           @local_port = random_local_port if @local_port.zero?
 
-          # First, grab a remote port
-          out, err, status = Open3.capture3(@env, *@ssh_cmd)
-          fail "Failed to request remote port: #{err}" unless status.success?
-          remote_port = out.chomp
-
-          # Then, spin up a SSH session using that port and port forwarding.
-          # Pass ExitOnForwardFailure to ensure nothing else can be listening
-          # on this port (thanks to Diego Argueta for reporting this issue).
-          tunnel_env = @env.merge(
-            'TUNNEL_PORT' => remote_port, # Request a specific port
-            'TUNNEL_SIGNAL_OPEN' => '1'   # Request signal when tunnel is up
-          )
-
-          # TODO: Dynamically compose SendEnv from tunnel_env
           tunnel_cmd = @ssh_cmd + [
-            '-L', "#{@local_port}:localhost:#{remote_port}",
-            '-o', 'SendEnv=TUNNEL_PORT',
-            '-o', 'SendEnv=TUNNEL_SIGNAL_OPEN',
+            '-L', "#{@local_port}:#{@socket_path}",
```

Comment:
I was thinking of doing something a little different actually:

- First, always warn if the CLI is used outside of a toolbelt context. 
- Second, warn again when people actually try to use db:tunnel and things fail.

What do you think? The first point might be a little heavy-handed considering that on recent OSes, this will actually work just fine (Xenial has the right SSH, so does latest OSX, etc.)